client/js/_author/components/assessments/question_types/audio_upload.jsx

```diff
@@ -20,6 +21,24 @@ export default class AudioUpload extends React.Component {
     return seconds;
   }
 
+  static rangeWarning() {
+    return (
+      <span className="author--c-inline-error">Please enter a positive number under 240</span>
+    );
+  }
+
+  constructor(props) {
+    super(props);
+    const timeLimit = AudioUpload.getAudioLimit(props.item)
+      || `${AudioUpload.MAX_TIME}`;
+    const displayWarning = timeLimit > AudioUpload.MAX_TIME;
```

Comment:
So you want this value as a string? You are then using `>` on strings. This could cause some funky edge cases. Take `"2">"10"` for example.